kzz7yh-f47354/cod-ve09v2_kzz7yh-f47354.xml

```diff
@@ -160,12 +161,13 @@
             </div>
             <div xml:id="kzz7yh-f47354-Dd1e285">
                <head xml:id="kzz7yh-f47354-Hd1e287">Quaestio 2</head>
+               <head type="question-title">Utrum deus diem septimus sanctificauerit</head>
                <p xml:id="kzz7yh-f47354-d1e289">
       Quaesio II. 
-      <lb ed="#V" n="106"/>Ecundo queritur vtrum deus diem 
-      <lb ed="#V" n="107"/>
+      <lb ed="#V" n="106"/>SEcundo queritur vtrum deus diem 
+      <lb ed="#V" n="107"/><!--TODO: error line break-->
                   <lb ed="#V" n="108"/>septimum sanctificauerit. Et 
-      <lb ed="#V" n="109"/>videtur quod non: quia qui denon potest recipere gratiam 
+      <lb ed="#V" n="109"/>videtur quod non: quia qui denom potest recipere gratiam 
```

Comment:
"denom" isn't correct but I am unsure of how to read this line

Reply:
@jeffreycwitt

Reply:
I think this is "quod non". 

I will make this change.